### ISSUE TITLE
Actually do something when the limit is reached 

### ISSUE THREAD

**Issue by Turhvjbufv** [sweakpl/unlock-master#2]
Instead of doing a notification, add an option to lock(allow calls only,almost all apps except basic ones will be locked)the phone for some time(maybe an hour, maybe all day).
Also please add the option for screen time limit, that will also have the option to lock the phone for some time when a certain threshold is reached. 
Thanks.

**Comment by sweakpl** [comment]
Hey, thank you for sugestions!

1. When it comes to locking the user away from their applications I think this would require some special system permissions that "normal" apps are not granted with, but I'm not sure at the moment. If it is possible then I would definitely consider adding this type of functionality,
2. Screen time limit is something I thought of as well, but I have decided to develop this feature sometime in the future.

**Comment by Turhvjbufv** [comment]
Yes I'm pretty sure that locking apps requires special permission but I would love to see this since there are no foss apps that can lock apps based on a limit for unlockes(and in the future also time spent)
(might be wrong but I'm pretty sure about that)
